modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
+[NOTE]
+Studio doesn’t currently support scaffolding API specs that reference OAS, JSON schema, or AsyncAPI fragments imported from Exchange as project dependencies. To scaffold a spec with these fragments, you must provide the fragments inline within the spec or through fragment files that you create manually within your project directory. This limitation doesn’t affect RAML fragments from Exchange, which are fully supported.
 
 == API Specifications as Local Copies in Your Projects
 
-You can still import a RAML API specification either xref:import-api-specification-local-file.adoc[from your hard drive] or xref:import-api-specification-design-center.adoc[directly from Design Center] into the `/src/main/api/` directory in the structure of your project. This process creates a local copy of the API in your project's structure and the imported API works as a standalone attachment to your project, meaning that every subsequent upgrade requires a manual import.
+Import a RAML API specification either xref:import-api-specification-local-file.adoc[from your hard drive] or xref:import-api-specification-design-center.adoc[directly from Design Center] into the `/src/main/api/` directory in the structure of your project. This process creates a local copy of the API in your project's structure. The imported API works as a standalone attachment to your project, meaning that every additional upgrade requires a manual import.
 
-Whenever the API that you import as a local copy into your project's structure is updated outside your Studio environment, you must reimport the API into your project and regenerate all the flows based on the updated specification, risking creating duplicated copies of your specification.
+When the API imported as a local copy into your project's structure is updated outside your Studio environment, reimport the API into your project. Studio regenerates all the flows based on the updated specification. 
 
-You can only keep RAML API specifications as local copies within your project's structure.
+[NOTE]
+Duplicated copies of your specifications can be created on your local drive. You can only keep RAML API specifications as local copies within your project's structure.
 
 == API Specifications as API Specification Projects in Studio
 
-You can start xref:create-api-specification-studio.adoc[an API specification project in Studio and sync it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
+Start xref:create-api-specification-studio.adoc[an API specification project in Studio and sync it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
```

Comment:
I'm wondering about all these in-line links. Is there a reason you're doing it this way instead of as a "See..."? It's just a lot of maintenance with the hard-coded text.

Reply:
Why is it harder to maintain the hard-coded text if it is in-line? The sentence reads fine because it does not depend on the title of the target article.

Reply:
I guess not. But, I guess the question is - do you want to send people to another document in the middle of a sentence?

Here are the guidelines:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-links

Reply:
OK! I think I got it now. Let's see if my rewording works.
Thanks!